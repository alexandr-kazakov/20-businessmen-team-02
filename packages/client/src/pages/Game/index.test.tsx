import React from 'react'
import { render } from '@testing-library/react'

import GamePage from '@/pages/Game/index'

jest.mock('react-redux', () => ({
  useSelector: () => ({ user: {} }),
  useDispatch: jest.fn(),
}))

test('should render GamePage component', () => {
  const { asFragment } = render(<GamePage />)

  expect(asFragment()).toMatchSnapshot()
})
