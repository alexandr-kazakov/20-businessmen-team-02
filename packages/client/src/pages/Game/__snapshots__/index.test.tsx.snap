// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GamePage component 1`] = `
<DocumentFragment>
  <div>
    <div>
      <canvas
        height="768"
        width="728"
      />
      <div
        class=""
      >
        <div>
          Для переключения в полноэкранный режим нажмите Ctrl-Q
        </div>
        <div
          id="dropZone"
        >
          <div>
            Вы можете перенести сюда файл с картинкой, которую будете собирать
          </div>
        </div>
        <label
          for="level"
        >
          Выберите уровень:
        </label>
        <select
          id="level"
        >
          <option
            value="0"
          >
            Новичок
          </option>
          <option
            value="1"
          >
            Продвинутый
          </option>
          <option
            value="2"
          >
            Эксперт
          </option>
        </select>
        <button
          class=""
        >
          Старт
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
